<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-10-07 10:33:16.932 ***</b>
=ERROR REPORT==== 7-Oct-2024::10:33:16.932374 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.244559.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1973}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1934}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-10-07 10:33:16.933 ***</b>
=CRASH REPORT==== 7-Oct-2024::10:33:16.932831 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.244565.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2426)
    ancestors: [<0.244559.0>]
    message_queue_len: 0
    messages: []
    links: [<0.244559.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 6882
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-10-07 10:33:56.930 ***</b>
=ERROR REPORT==== 7-Oct-2024::10:33:56.930702 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.248908.0&gt;,
                               {#Ref&lt;0.3818106469.3591897089.37467&gt;,1005936,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.3818106469.3591897089.37467&gt;,1005936,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-10-07 10:33:56.931 ***</b>
=CRASH REPORT==== 7-Oct-2024::10:33:56.930990 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.248929.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.3818106469.3591897089.37467&gt;,1005936,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [&lt;0.248908.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4511
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init_local_registered_parent/1 2024-10-07 10:33:56.953 ***</b>
=CRASH REPORT==== 7-Oct-2024::10:33:56.953768 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_local/2
    pid: &lt;0.248934.0&gt;
    registered_name: my_server
    exception exit: {&lt;0.248932.0&gt;,stopped}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [foobar]
    message_queue_len: 0
    messages: []
    links: [&lt;0.248932.0&gt;]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 3454
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-10-07 10:38:32.087 ***</b>
=CRASH REPORT==== 7-Oct-2024::10:38:32.086991 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.254726.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.3818106469.3593207809.211391&gt;,5656,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1694)
    ancestors: [&lt;0.254724.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28124
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-07 10:38:41.515 ***</b>
=ERROR REPORT==== 7-Oct-2024::10:38:41.515059 ===
** Generic server <0.256004.0> terminating 
** Last message in was {nodedown,'peer_down_crash-388517-11741@ae15bca87445'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-388517-11741",
                              peer_down => crash},
                            'peer_down_crash-388517-11741@ae15bca87445',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,49,46,49,47,98,
                             105,110,47,101,114,108],
                            ["-sname","peer_down_crash-388517-11741",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQGFlMTViY2E4NzQ0NQAD6AQAAAAAZwOaeA=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-388517-11741@ae15bca87445'}},
                            {<0.255993.0>,
                             #Ref<0.3818106469.3585343492.171382>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-388517-11741@ae15bca87445'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-10-07 10:38:41.515 ***</b>
=CRASH REPORT==== 7-Oct-2024::10:38:41.515631 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256004.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-388517-11741@ae15bca87445'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [<0.255993.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11328
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-07 10:38:42.616 ***</b>
=WARNING REPORT==== 7-Oct-2024::10:38:41.614346 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_stop-388741-11741@ae15bca87445' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-07 10:38:42.654 ***</b>
=WARNING REPORT==== 7-Oct-2024::10:38:41.654212 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_halt-388709-11741@ae15bca87445' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-07 10:38:43.141 ***</b>
=WARNING REPORT==== 7-Oct-2024::10:38:43.141442 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-388677-11741@ae15bca87445'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-07 10:38:43.200 ***</b>
=WARNING REPORT==== 7-Oct-2024::10:38:43.200351 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-551586-11741@ae15bca87445'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-10-07 10:38:58.735 ***</b>
=WARNING REPORT==== 7-Oct-2024::10:38:57.733359 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-552098-11741@ae15bca87445' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-10-07 10:38:59.256 ***</b>
=WARNING REPORT==== 7-Oct-2024::10:38:59.256428 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-552066-11741@ae15bca87445'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-10-07 10:38:59.261 ***</b>
=WARNING REPORT==== 7-Oct-2024::10:38:59.261317 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-389125-11741@ae15bca87445'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-07 10:39:01.505 ***</b>
=ERROR REPORT==== 7-Oct-2024::10:39:01.503690 ===
** Generic server <0.256377.0> terminating 
** Last message in was {'EXIT',<0.256374.0>,
                               {#Ref<0.3818106469.3585867778.258965>,1659935,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-385508-11741",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-385508-11741@ae15bca87445',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,49,46,49,47,98,105,110,47,101,114,
                                     108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-385508-11741",
                                     "-user","peer"],
                                    #Port<0.3126>,undefined,<<>>,running,
                                    {<0.256374.0>,
                                     #Ref<0.3818106469.3585605635.58431>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.3818106469.3585867778.258965>,1659935,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-07 10:39:01.507 ***</b>
=CRASH REPORT==== 7-Oct-2024::10:39:01.506895 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256377.0>
    registered_name: []
    exception exit: {#Ref<0.3818106469.3585867778.258965>,1659935,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.256374.0>]
    message_queue_len: 2
    messages: [{#Port<0.3126>,
                   {data,<<"=SUPERVISOR REPORT==== 7-Oct-2024::10:39:01.346312 ===\n    supervisor: {<0.72.0>,user_sup}\n    errorContext: child_terminated\n    reason: killed\n    offender: [{pid,<0.73.0>},{mod,user_sup}]\n\nFailed to write log message to stdout, trying stderr\n=ERROR REPORT==== 7-Oct-2024::10:39:01.445156 ===\n** Generic server <0.72.0> terminating \n** Last message in was {'EXIT',<0.73.0>,killed}\n** When Server state == {state,user_sup,undefined,<0.73.0>,\n                               {<0.72.0>,user_sup}}\n** Reason for termination ==\n** killed\n\nFailed to write log message to stdout, trying stderr\n=CRASH REPORT==== 7-Oct-2024::10:39:01.346397 ===\n  crasher:\n    initial call: peer:init_supervision/2\n    pid: <0.71.0>\n    registered_name: peer_supervision\n    exception exit: {peer_channel_terminated,killed}\n      in function  peer:loop_supervision/1 (peer.erl, line 1303)\n      in call from peer:init_supervision/2 (peer.erl, line 1281)\n    ancestors: [kernel_sup,<0.47.0>]\n    message_queue_len: 0\n    messages: []\n    links: [<0.49.0>]\n    dictionary: []\n    trap_exit: true\n    status: running\n    heap_size: 376\n    stack_size: 29\n    reductions: 80\n  neighbours:\n\nFailed to write log message to stdout, trying stderr\n=CRASH REPORT==== 7-Oct-2024::10:39:01.445328 ===\n  crasher:\n    initial call: supervisor_bridge:user_sup/1\n    pid: <0.72.0>\n    registered_name: []\n    exception exit: killed\n      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)\n    ancestors: [kernel_sup,<0.47.0>]\n    message_queue_len: 0\n    messages: []\n    links: [<0.49.0>]\n    dictionary: []\n    trap_exit: true\n    status: running\n    heap_size: 2586\n    stack_size: 29\n    reductions: 6370\n  neighbours:\n\nFailed to write log message to stdout, trying stderr\n=SUPERVISOR REPORT==== 7-Oct-2024::10:39:01.501661 ===\n    supervisor: {local,kernel_sup}\n    errorContext: child_terminated\n    reason: {peer_channel_terminated,killed}\n    offender: [{pid,<0.71.0>},\n               {id,peer_supervision},\n               {mfargs,{peer,start_supervision,[]}},\n               {restart_type,permanent},\n               {significant,false},\n               {shutdown,1000},\n               {child_type,worker}]\n\nFailed to write log message to stdout, trying stderr\n=SUPERVISOR REPORT==== 7-Oct-2024::10:39:01.501810 ===\n    supervisor: {local,kernel_sup}\n    errorContext: shutdown\n    reason: reached_max_restart_intensity\n    offender: [{pid,<0.71.0>},\n               {id,peer_supervision},\n               {mfargs,{peer,start_supervision,[]}},\n               {restart_type,permanent},\n               {significant,false},\n               {shutdown,1000},\n               {child_type,worker}]\n\n">>}},
                  {'EXIT',#Port<0.3126>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 10657
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-07 10:39:02.247 ***</b>
=ERROR REPORT==== 7-Oct-2024::10:39:02.246706 ===
** Generic server <0.256397.0> terminating 
** Last message in was {'EXIT',<0.256358.0>,
                           {#Ref<0.3818106469.3585867778.258944>,2409266,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "EJTZOUOAWLNWVCXIDSYG","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@ae15bca87445.2024-10-07_10.23.19/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-385476-11741",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@ae15bca87445.2024-10-07_10.23.19/make_test_dir.stdlib_test.logs/run.2024-10-07_10.23.55/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-385476-11741",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5.3/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-385476-11741@ae15bca87445',
                                    "/usr/local/lib/erlang-26.2.5.3/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-385476-11741",
                                     "-setcookie","EJTZOUOAWLNWVCXIDSYG",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@ae15bca87445.2024-10-07_10.23.19/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-385476-11741",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@ae15bca87445.2024-10-07_10.23.19/make_test_dir.stdlib_test.logs/run.2024-10-07_10.23.55/log_private/",
                                     "-user","peer"],
                                    #Port<0.3146>,undefined,<<>>,running,
                                    {<0.256358.0>,
                                     #Ref<0.3818106469.3585343492.174982>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.3818106469.3585867778.258944>,2409266,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-07 10:39:02.247 ***</b>
=CRASH REPORT==== 7-Oct-2024::10:39:02.247530 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256397.0>
    registered_name: []
    exception exit: {#Ref<0.3818106469.3585867778.258944>,2409266,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.256358.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3146>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 10958
    stack_size: 29
    reductions: 20918
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-07 10:39:02.439 ***</b>
=WARNING REPORT==== 7-Oct-2024::10:39:01.437252 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-385380-11741@ae15bca87445' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-07 10:39:02.742 ***</b>
=WARNING REPORT==== 7-Oct-2024::10:39:02.742472 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-389381-11741@ae15bca87445'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-07 10:39:03.053 ***</b>
=WARNING REPORT==== 7-Oct-2024::10:39:03.053413 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-385412-11741@ae15bca87445'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-07 10:39:03.093 ***</b>
=WARNING REPORT==== 7-Oct-2024::10:39:03.093260 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-385348-11741@ae15bca87445'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-07 10:50:14.484 ***</b>
=ERROR REPORT==== 7-Oct-2024::10:50:14.484654 ===
** Generic server <0.303158.0> terminating 
** Last message in was {'EXIT',<0.303155.0>,
                               {#Ref<0.3818106469.3603955714.135463>,351118,
                                [{child1,<0.303164.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.303158.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.303164.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459138],
                            1,0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.3818106469.3603955714.135463>,351118,
    [{child1,<0.303164.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-07 10:50:14.485 ***</b>
=ERROR REPORT==== 7-Oct-2024::10:50:14.484654 ===
** Generic server <0.303157.0> terminating 
** Last message in was {'EXIT',<0.303155.0>,
                               {#Ref<0.3818106469.3603955714.135463>,351118,
                                [{child1,<0.303164.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.303157.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.303163.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459138],
                            1,0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.3818106469.3603955714.135463>,351118,
    [{child1,<0.303164.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-07 10:50:14.485 ***</b>
=CRASH REPORT==== 7-Oct-2024::10:50:14.485187 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.303158.0>
    registered_name: []
    exception exit: {#Ref<0.3818106469.3603955714.135463>,351118,
                     [{child1,<0.303164.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.303155.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 14887
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-07 10:50:14.485 ***</b>
=CRASH REPORT==== 7-Oct-2024::10:50:14.485293 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.303157.0>
    registered_name: []
    exception exit: {#Ref<0.3818106469.3603955714.135463>,351118,
                     [{child1,<0.303164.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.303155.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 14960
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during zzz_SUITE:lc_graph/1 2024-10-07 11:16:19.970 ***</b>
=WARNING REPORT==== 7-Oct-2024::11:16:19.969571 ===
Process <0.406812.0> on node 'test_server@ae15bca87445' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Mon Oct 07 2024 10:23:55<!--/date--><br />
</div>
</center>
</body>
</html>
